source/upgrade.txt

```diff
@@ -66,6 +66,19 @@ and upgrade versions.
   If you're upgrading the {+driver-short+} from version 3.0 to version 3.10,
   address all breaking changes listed for versions 3.1 to 3.10, if any.
 
+.. _version-4.0-breaking-changes:
+
+Version 4.0 Breaking Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Drops support for the Boost and MNMLSTC/core external polyfill libraries. You can no
```

Comment:
S: add this line from the earlier section (looks like it's missing in the 3.11 section also)
```suggestion
~~~~~~~~~~~~~~~~~~~~~~~~~~~~

This driver version introduces the following breaking changes:

- Drops support for the Boost and MNMLSTC/core external polyfill libraries. You can no
```